bulletroboy/exoforce.py

```diff
@@ -286,3 +294,122 @@ def get_motors(self):
 
 	def get_tendons(self):
 		return [ muscle.tendon for muscle in self.muscle_units ]
+
+
+class Movements():
```

Comment:
- [ ] add docsting to every function in this class, make sure to describe the input parameters. everything except for init and get_EF is not obvious